com.unity.render-pipelines.universal/Runtime/Passes/CopyColorPass.cs

```diff
@@ -118,11 +141,14 @@ public override void OnCameraCleanup(CommandBuffer cmd)
             if (cmd == null)
                 throw new ArgumentNullException("cmd");
 
-            if (destination != RenderTargetHandle.CameraTarget)
```

Comment:
I assume `CopyColorPass` can be setup with `RenderTargetIdentifier` which I have also marked as obsolete.
This can be done with user made passes.
In order to remain backwards compatible for at least one revision, I need to support that case.
Having marked `destinationID` as obsolete, I have to disable it in this block which itself is obsolete.

I will add a note there.